manage-config

```diff
@@ -58,88 +67,113 @@ function get_section_opts(){
-                if [ ! "$s" = "$v" ]; then
-                    ret=2
-                    echo "Option $n should be set to [$v] instead of [$s]"
+        # Verify that the kernel options we want to add are now in the updated configuration
+        if [ -f ${inclusion_file} ]; then
+            echo
+            echo "Checking added kernel options..."
+            while read -r opt; do
+                if [ ! -z "$opt" ] && [[ ! "$opt" =~ ^#.* ]]; then
+                    n=${opt%=*}
+                    v="${opt#*=}"
+                    v="${v/#\"/}"
+                    v="${v/%\"/}"
+                    s=$(scripts/config --file ${CONFIG_FILE} -k --state $n)
+                    if [ ! "$s" = "$v" ]; then
+                        ret=2
+                        echo "Option $n should be set to [$v] instead of [$s]"
+                    fi
                 fi
+            done <<< ${inclusion_opts};
+            if [ ! $ret = 2 ]; then
+                echo "No error"
             fi
-        done <<< ${inclusion_opts};
-        if [ ! $ret = 2 ]; then
-            echo "No error"
         fi
-    fi
 
-    # Process any force inclusions in the kernel
-    if [ -f ${force_inclusion_file} ]; then
-        force_inclusion_opts=$(get_section_opts ${force_inclusion_file} "common" ${ARCH} ${PLATFORM})
-        while read -r opt; do
-            if [ ! -z "$opt" ] && [[ ! "$opt" =~ ^#.* ]]; then
-                echo $opt >> ${CONFIG_FILE}
-            fi
-        done <<< ${force_inclusion_opts};
+        # Process any force inclusions in the kernel
+        if [ -f ${force_inclusion_file} ]; then
+            force_inclusion_opts=$(get_section_opts ${force_inclusion_file} "common" ${ARCH} ${PLATFORM})
+            while read -r opt; do
+                if [ ! -z "$opt" ] && [[ ! "$opt" =~ ^#.* ]]; then
+                    echo $opt >> ${CONFIG_FILE}
+                fi
+            done <<< ${force_inclusion_opts};
+        fi
+
+        echo
     fi
 
-    echo
+    echo "process_inclusion_exclusion_files Done"
+    return $ret
+}
+
+exclusion_file="../patch/kconfig-exclusions"
+inclusion_file="../patch/kconfig-inclusions"
+force_inclusion_file="../patch/kconfig-force-inclusions"
+ret_process_inc_ex=0
+ret_process_inc_ex=$(process_inclusion_exclusion_files > /dev/null; echo $?)
+
+#  Secure Boot support
+if [ $ret_process_inc_ex -e 0 ]; then
+    if [ ${SECURE_UPGRADE_MODE} == "dev" -o ${SECURE_UPGRADE_MODE} == "prod" ]; then
+        echo "set kconfig-secure-boot-exclusions & kconfig-secure-boot-inclusions"
+        exclusion_file="../patch/kconfig-secure-boot-exclusions"
+        inclusion_file="../patch/kconfig-secure-boot-inclusions"
+        force_inclusion_file="../patch/kconfig-force-secure-boot-inclusions"
+
+        # save the new pub key in kernel
+        sed -i "s|^CONFIG_SYSTEM_TRUSTED_KEYS=.*|CONFIG_SYSTEM_TRUSTED_KEYS=\"$SECURE_UPGRADE_DEV_SIGNING_CERT\"|g" $inclusion_file
```

Comment:
include ${inclusion_file} into the common double quotes

Reply:
its working without {} , but I agree to saved the convention